machines/bleShare/scan/scanMachine.ts

```diff
@@ -595,12 +594,9 @@ export const scanMachine =
             accepted: {
               entry: ['logShared', 'sendVcShareSuccessEvent'],
               on: {
-                DISMISS: {
+                DISCONNECT: {
```

Comment:
@Anil-kumar-Majji Don't rename DISMISS by DISCONNECT as DISMISS event is being used from other places.
Apart from this, inner state should be disconnect instead of navigatingToHome as we are not navigating to home